static/app/components/gridEditable/styles.tsx

```diff
@@ -242,7 +242,6 @@ const GridStatusFloat = styled('div')`
   width: 100%;
   height: ${GRID_STATUS_MESSAGE_HEIGHT}px;
   overflow: hidden;
-
```

Comment:
we can remove this change to keep git history clean

Reply:
🫡